models/models.go

```diff
@@ -207,32 +207,6 @@ type BuildRunStartModel struct {
 	StartTime   time.Time `json:"start_time" yaml:"start_time"`
 }
 
-type BuildRunResultsModel struct {
```

Comment:
The data models and related functions are moved to `models/build_run_results.go`.